test/helpers/IntegrationHelper.js

```diff
@@ -278,6 +277,37 @@ var customMatchers = {
             }
         };
     },
+    toHaveClaimInitiativeAbilityButton: function (util, customEqualityMatchers) {
```

Comment:
We need to add this to the wiki once merged, lmk if you're down to do it or else I'm happy to. Ditto for toHaveInitiative and player.claimInitiative().

Reply:
Ah right, I can add it once it's merged.